recipes/tepid/meta.yaml

```diff
@@ -0,0 +1,41 @@
+about:
+  home: 'https://github.com/ListerLab/TEPID'
+  license: 'GPL'
+  summary: 'TEPID uses paired-end illumina sequencing reads to identify novel TE variants.'
+
+build:
+  number: 0
+  skip: True # [not py27 or osx]
+
+package:
+  name: tepid
+  version: '1.0'
+
+requirements:
+  build:
+      - gcc
+      - zlib
+      - python
+      - 'pysam <0.9,>0.8'
+      - pybedtools
+      - 'samtools >1.1,<1.3'
+      - bedtools 2.25.0
+      - numpy 1.9.2
+      - pandas
+      - nose
+  run:
+      - python
+      - bowtie2
+      - 'samtools >1.1,<1.3'
+      - 'samblaster >=0.1.19'
+      - bedtools 2.25.0
+      - yaha
+
+source:
+  git_url: 'https://github.com/ListerLab/TEPID.git'
```

Comment:
Can you use https://github.com/ListerLab/TEPID/archive/0.7.tar.gz instead, or any other release or checksum. Only using the git url without any version number will just take the latest version, which is not reproducible. Also whenever you can please take a tarball instead of a git_url.